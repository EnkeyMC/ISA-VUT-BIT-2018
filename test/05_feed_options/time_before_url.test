NAME=Show time (param -T before URL)
ARGS=-T http://www.mocky.io/v2/5bd3096a3400003600cfdff9
RC=0
CHECK_OUTPUT=1
