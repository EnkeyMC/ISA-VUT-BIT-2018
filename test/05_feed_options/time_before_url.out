*** Scripting News ***
Law and Order
Aktualizace: Sun, 29 Sep 2002 23:48:33 GMT

Rule 1
Aktualizace: Sun, 29 Sep 2002 17:24:20 GMT

Really early morning no-coffee notes
Aktualizace: Sun, 29 Sep 2002 11:13:10 GMT